packages/variable-editor/src/utils/string/string.ts

```diff
@@ -0,0 +1,7 @@
+export const removeLeadingWhitespacesFromEachLine = (string: string) => {
+  return string.replace(/^[^\S\n]+/gm, '');
+};
+
+export const getLastLine = (string: string) => {
+  return string.split('\n').at(-1);
```

Comment:
I think there is a util function in the ui-components splitNewLine or something like that which also consider \r.